### PR TITLE
tests/trust: Add ipaadmin_password to test playbooks.

### PR DESCRIPTION
**Pull request by rjeffman**
Test playbooks were missing ipaadmin_password.

### CODE REVIEW

File: tests/trust/test_trust.yml [hunk 1]
```diff
@@ -36,10 +37,13 @@
 
     - name: add trust
       ipatrust:
+        ipaadmin_password: SomeADMINpassword
         realm: windows.local
         admin: Administrator
         password: secret_ad_pw
         state: present
+      register: result
```
Comment (t-woerner) [inline tests/trust/test_trust.yml:45]: This is unexpected with the commit title.

File: tests/trust/test_trust.yml [hunk 2]
```diff
@@ -48,4 +52,12 @@
       register: check_add_trust
       failed_when: "'1 trust matched' not in check_add_trust.stdout"
 
-    when: trust_test_is_supported | default(false)
\ No newline at end of file
+    - name: delete trust
```
Comment (t-woerner) [inline tests/trust/test_trust.yml:55]: This is unexpected with the commit title.

File: tests/trust/test_trust.yml (hunk 1 above)
Comment (rjeffman) [reply tests/trust/test_trust.yml]: This is another change, which is not ready yet. Sorry for that.

File: tests/trust/test_trust.yml (hunk 2 above)
Comment (rjeffman) [reply tests/trust/test_trust.yml]: fixed.

**Review by t-woerner — APPROVED** [review]
LGTM

**Pull request merged**